examples/4.cson

```diff
@@ -0,0 +1,35 @@
+
+chain :
+  user : "zapu"
+  ctime : "1000"
+  expire_in : 10000000
+  links : [
+    {
+      # ctime implied from chain ctime = 1000
+
+      type : "eldest"
+      label : "e"
+
+      key : {
+        gen : "pgp_ecc"
+
+        # Will result in generation time of 1200, but not affect
```

Comment:
`generated` (line 19) is "+200" so it takes "current clock" (which is set to 1000 because of `ctime` on line 4) and adds 200 to it, but it does not advance "current clock".